docs: remove image with caption and tabs extended media pages (#1112)
carbon-design-system/carbon-for-ibm-dotcom-website: src/pages/components/image.mdx

</Column>
</Row>
<Caption>When an image has a caption, the user can hover over the image and click to trigger a Lightbox overlay with more information.</Caption>

### Resources for Image with caption

<CardGroup>
  <MiniCard
    title="Design specifications"
    href="https://ibm.ent.box.com/folder/109643123498?s=mclmewshqcme7g6gykepok78rm882a0j"
  >
    <img src={sketchSymbol} alt="Sketch"/>
  </MiniCard>
  <MiniCard
    title="Functional specifications"
    href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Image-with-caption"
  >
    <img src={githubSymbol} alt="Github"/>
  </MiniCard>
  <MiniCard
    title="React Storybook"
    href="https://www.ibm.com/standards/carbon/react/?path=/story/components-image-with-caption--default"
  >
    <img src={reactSymbol} alt="React"/>
  </MiniCard>
  <MiniCard
    title="Web Components Storybook"
    href="https://www.ibm.com/standards/carbon/web-components/?path=/story/components-image-with-caption--default"
  >
    <img src={webComponentsSymbol} alt="Web Components"/>
  </MiniCard>
</CardGroup>

### Content guidance for Image with caption

> Image is a foundational component and doesn't contain further content than itself. Image with caption allows for more content to be authored. Please follow below content guidance.

Reviewer: ```suggestion
Image is a foundational component and does not contain any additional content. Image with caption provides the opportunity for additional content, as shown in the following table.
```

We recommend displaying images at 1x1, 2x1, 4x3, and 16x9 aspect ratios. For guidance on choosing or creating an IBM image, see the <a href="https://www.ibm.com/design/language/photography/overview" target="_blank">photography guidelines</a> on the IBM Design Language site. 

<Row>
<Column colMd={8} colLg={8}>

![Image of Image default in context](../../images/component/image/image-overview-default.jpg)

</Column>
</Row>

## Options

### Caption and zoom

An image can include a caption beneath the image itself to provide more context. With the caption feature, the image can also be clicked to open an overlay (Lightbox media viewer) with a larger version of the image. 

Inside the viewer, the caption becomes the heading and there is an opportunity for additional body text.

For more about Image with caption, see the <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Image-with-caption" target="_blank">functional specs</a>.

<Row>

<Column colMd={4} colLg={4}>

![Example image: single selection with no selected filter.](../../images/component/image/image-options-caption-hover.jpg)

> <Caption>The image with a caption displayed in context</Caption>

Reviewer: ```suggestion
<Caption>Example of an image with a caption in context</Caption>
```

</Column>

<Column colMd={4} colLg={4}>

![Example image: single selection with selected filter.](../../images/component/image/image-options-caption-lightbox.jpg)

> <Caption>Once clicked, the Lightbox media viewer displays a larger version of the image</Caption>

Reviewer: ```suggestion
<Caption>Once clicked, the Lightbox media viewer displays a larger version of the image.</Caption>
```

</Column>

</Row>

### Border

> An image can include a 1px #E0E0E0 ($decorative-01) border to help frame the images better when the page background and image background colors are the same.

Reviewer: Check with @shixiedesign. As a rule, we should probably only include the token here, to encourage connection to the system.

<Row>
<Column colMd={8} colLg={8}>

![Image of Image default in context](../../images/component/image/image-options-border.png)

</Column>
</Row>

> <Caption>The border separates the image from the background with the same color</Caption>

Reviewer: ```suggestion
<Caption>The border helps to separate the image from the background with the same color.</Caption>
```

> ## Content guidance

Reviewer: @RichKummer wondering if we get Jonathan to work on this table to get the updated version for this PR. Question for @shixiedesign. It probably wouldn't take him too long.
